ksp_plugin_adapter/config_node_parsers.cs

```diff
@@ -22,9 +24,11 @@ internal static class ConfigNodeParsers {
         axis_declination        = node.GetAtMostOneValue("axis_declination"),
         reference_angle         = node.GetAtMostOneValue("reference_angle"),
         angular_frequency       = node.GetAtMostOneValue("angular_frequency"),
-        reference_radius        = node.GetAtMostOneValue("reference_radius"),
-        j2                      = node.GetAtMostOneValue("j2"),
-        geopotential            = node.GetBodyGeopotentialElements().ToArray()
+        reference_radius =
+            node?.GetAtMostOneValue("reference_radius") ??
+            (j2 != null || geopotential != null ? body.Radius + " m" : null),
```

Comment:
Keep it required for `NewCartesianBodyParameters`, i.e., revert the changes in this function.
See the comment about *nominal* vs. *sufficient* in the documentation: `NewCartesianBodyParameters` is the *sufficient* path.